xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -729,8 +729,8 @@ In application code, this field is most commonly used to safely represent a <xre
         <Parameter Name="right" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
+        <param name="left">The <xref data-throw-if-not-resolved="true" uid="Microsoft.Extensions.Primitives.StringValues"></xref> to compare.</param>
+        <param name="right">The <xref data-throw-if-not-resolved="true" uid="System.String"></xref> to compare.</param>
```

Comment:
Ugh I see what you mean, this is wrong. Unsure what's happening. You can use plain english for these:

```suggestion
        <param name="left">The string values to compare.</param>
        <param name="right">The string to compare.</param>
```